Service/ContentTypeService.php

```diff
@@ -409,4 +417,57 @@ public function getAllTypes()
         $this->loadEnvironment();
         return implode(',', array_keys($this->contentTypeArrayByName));
     }
+    
+    public function readJson(File $jsonFile, Environment $environment): ContentType
+    {
+        $fileContent = file_get_contents($jsonFile->getRealPath());
+       
+        $encoders = array(new JsonEncoder());
+        $normalizers = array(new JsonNormalizer());
+        $serializer = new Serializer($normalizers, $encoders);
+        /**@var ContentType $contentType */
+        $contentType = $serializer->deserialize(
+            $fileContent,
+            "EMS\CoreBundle\Entity\ContentType",
+            'json'
+        );
+        $contentType->setEnvironment($environment);
+        return $contentType;
+    }
+    
+    public function createContentType(ContentType $contentType): ContentType
```

Comment:
shorter named methods are better (if they are still clear on their purpose). We are in the ContentTypeService, so "create" is a good function name

Reply:
After reading the code, I see we change the content type that is provided, and we persist it.
To reflect the business of this function I would vote for "persistAsNew"